### ISSUE TITLE
Rename default branch from "master" to "main"

### ISSUE THREAD

**Issue by segheysens** [twilio/twilio-voice.js#83]
**Is your feature request related to a problem? Please describe.**
Related to social problems.

**Describe the solution you'd like**
For inclusivity and following Twilio's goal of working towards being an anti-racist company, I believe it's in line with our values to rename our master branch to "main".

**Describe alternatives you've considered**
Haven't considered other branch naming alternatives.

**Additional context**
This may require an audit of known dependencies on a "master" branch. There's [a short article here](https://www.theserverside.com/feature/Why-GitHub-renamed-its-master-branch-to-main) with more context, but the long and short is that "master and slave" naming paradigms in tech are outdated, racist conventions.

**Comment by xsteviex** [comment]
Hi @segheysens, I don't particularly have a problem with renaming the branch "main" or "primary" or any other "master" synonym. However, I do find the reasoning for this change to be quite concerning. As a minority in tech, I personally dislike the feeling that I'm being pandered to. We understand the context of what master means, and don't need to be shielded from the word. If you want to rename the branch, it's fine, however, it's not bringing about social change. It's just grandstanding.

**Comment by segheysens** [comment]
Hey @xsteviex, thanks for the feedback, I really appreciate you taking the time to explain.

It wasn't my intention to pander/grandstand and you made a solid point about how understanding the context of "master", in this regard as a developer, is different - I just had a great discussion with a Black coworker who agreed about the importance of context, and we chatted about how individual experiences are often going to play into how somebody feels about this (in their view, this change didn't feel that this change would do much for them, from a social perspective). Obviously, their POV also doesn't account for everyone's experiences, but it was helpful input. I had seen accounts from others members in the Black tech community who have shared the opinion that this change is impactful to their experience - that's the extent of what I had in mind when I opened the issue.

The caveat my coworker and I chatted about was whether Twilio having some repos with a "master" branch, and others with a "main" branch, could cause confusion for future engineers contributing to the projects/products over here; as opposed to all of our repos using the same terminology. I think this is where you were going with your first point about not minding whether the branch is renamed to a synonym.

Happy to close this issue for now - if that is the case, I'm sure that somebody else will open a "renaming the branch" issue in the future.
